ja2003.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.christianrauch.Jahresarbeit-2003</id>
+  
+  <name>Jahresarbeit 2003</name>
+  <summary>Find die größte Fläche von zusammenhängenden Quadraten</summary>
```

Comment:
I need to agree with @hfiguiere here. Currently, we expect the default description to be in English, as we are unable to guarantee availability of translations for all applications. You can do something like `<description xml:lang="de">` (the same for summary) and it will be correctly interpreted by native frontends like GNOME Software. I'd also include a comment that this application is available only in German.

Reply:
This works for `<summary xml:lang="de">` and `<summary xml:lang="en">` but `<description xml:lang="de">` gives me the error: `com.github.christianrauch.Jahresarbeit-2003: AppData problem: tag-missing : <description> required`. So, I am just doing the translation for the `summary`.

Reply:
I believe you can simply skip `xml:lang` for English altogether and leave it without any tag. This should fix the problem for the description (and I'd drop it from the summary as well).

Reply:
Ok, this seems to work. Only one of the `description` tags must be without the `xml:lang`. I now have `summary` and `description` in both languages.